examples/onepanel_submitter.py

```diff
@@ -0,0 +1,140 @@
+                api_response = api_instance.create_workflow_template_version(namespace, workflow_template_uid, body)
+                logging.info("New template version added to %s" % workflow_template_uid)
+                print(api_response)
+            except ApiException as e:
+                print("Exception when calling WorkflowTemplateServiceApi->create_workflow_template_version: %s\n" % e)
+
+    def _execute_workflow(self, namespace, workflow_template_uid):
+        params = []
+        # Enter a context with an instance of the API client
+        with onepanel.core.api.ApiClient(self.configuration) as api_client:
+            # Create an instance of the API class
+            api_instance = onepanel.core.api.WorkflowServiceApi(api_client)
+            # NOTE: `label` was added here
+            body = onepanel.core.api.CreateWorkflowExecutionBody(
+                parameters=params,
+                workflow_template_uid=workflow_template_uid,
+                labels=[{'key': 'created-by', 'value': 'couler'}])
+            try:
+                workflow = api_instance.create_workflow_execution(namespace, body)
+                if workflow:
+                    logging.info('Executing Workflow %s in namespace: %s' % (workflow.name, self.namespace))
+            except ApiException as e:
+                print('There was an error executing the Workflow: %s\n' % e)
+
+    def _get_token(self, host, username, token):
+        with onepanel.core.api.ApiClient(self.configuration) as api_client:
+            # Create an instance of the API class
+            api_instance=onepanel.core.api.AuthServiceApi(api_client)
+            body=onepanel.core.api.GetAccessTokenRequest(username, token) # LogInRequest
+
+            try:
+                # NOTE: This should use the new login command
+                return api_instance.get_access_token(body).to_dict()['access_token']
+
+            except ApiException as e:
+                print('There was an error with your credentials: %s\n' % e)
+
+    def template_to_argo(self, workflow_yaml):
+        output_dict = json.loads(json.dumps(workflow_yaml))
+        yaml = output_dict['spec']
+        manifest = pyaml.dump(yaml)
+        return manifest
+
+    def submit(self, workflow_yaml, secrets=None):
+        if secrets:
+            for secret in secrets:
+                self._create_secret(secret.to_yaml())
+        # NOTE: Need to create or update
+        # Need to first get Workflow Template: https://github.com/onepanelio/python-sdk/blob/master/docs/WorkflowTemplateServiceApi.md#get_workflow_template
+        # If the Workflow Template doesn't exist, then create the Workflow Template like we do below
+        # If the hashes are different, create a new template version: https://github.com/onepanelio/python-sdk/blob/master/docs/WorkflowTemplateServiceApi.md#create_workflow_template_version
+        workflow_template=self._create_workflow_template(workflow_yaml, self.workflow_name)
```

Comment:
The method calls should all be here as follows:
- `_get_workflow_template`
- If `workflow_template` exists, call `_create_worfklow_template_version`
- If it does not exist, call `create_workflow_template`